options/locale/locale_en-US.ini

```diff
@@ -991,7 +991,8 @@ template.issue_labels = Issue Labels
 template.one_item = Must select at least one template item
 template.invalid = Must select a template repository
 
-archive.title = This repo is archived. You can view files and clone it, but cannot push or open issues/pull-requests.
+archive.title_no_date = This repo is archived. You can view files and clone it, but cannot push or open issues/pull-requests.
```

Comment:
Why not stay with `archive.title`?

Reply:
Agree we had better to stay with old title, then translators do not need to re-translate this unchanged string.

Reply:
Yeah, this is a pointless rename of a translation key. Apparently it already takes months to get some languages updated, so we need to be much more careful around translations, re-use where possible etc.

Reply:
The change was requested in another Review, but I have now changed it back.